bitrise.yml

```diff
@@ -1547,6 +1549,7 @@ workflows:
 
             FIREBASE_TEST_RESULT=$($HOME/google-cloud-sdk/bin/gcloud firebase test ios run \
                 --test "$TARGET_ARCHIVE" \
+                --xcode-version=16.2 \
```

Comment:
The default version of Xcode on Firebase is an older, so we need to specify 16.2 explicitly.